assets/src/components/nav/topNavMobile.tsx

```diff
@@ -13,10 +13,28 @@ import {
 } from "../../helpers/icon"
 import { toggleMobileMenu, toggleNotificationDrawer } from "../../state"
 import NotificationBellIcon from "../notificationBellIcon"
-import { pageOrTabName } from "../../models/routeTab"
+import { currentTabName, RouteTab } from "../../models/routeTab"
 import { openDrift } from "../../helpers/drift"
 import { reload } from "../../models/browser"
 
+export const toTitleCase = (str: string): string => {
+  return str.replace(
+    /\w\S*/g,
+    (txt: string): string =>
+      txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
+  )
+}
+
+export const pageOrTabName = (location: any, routeTabs: RouteTab[]): string => {
```

Comment:
I think it's possible to give `location` a more specific type than `any` here - see `src/helpers/appCue.ts` for an example.

Reply:
Resolved in eb41669 - also in the process I moved a function to create a test Location out of the appCue test and into a test helper.